paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/UsBankAccountFormArgumentsKtx.kt

```diff
@@ -61,7 +62,7 @@ private fun USBankAccountFormArguments.updatePrimaryButton(
 internal fun USBankAccountFormArguments.updateMandateText(
     context: Context,
     screenState: USBankAccountFormScreenState,
-    mandateText: String?,
+    mandateText: ResolvableString?,
```

Comment:
Can we add concatenated resolvable strings to avoid having to pass a `Context` here? Not in this pull request, but eventually.

Reply:
Yup. The big change we'd need to make is to make the transforms consider replacements as being resolvable strings instead of just strings. And we'd need to feed the context through to be able to do that.

It's not on my TODO list for now, but should be doable.